src/v/utils/to_string.h

```diff
@@ -12,13 +12,29 @@
 #pragma once
 
 #include "seastarx.h"
+#include "ssx/sformat.h"
 
 #include <seastar/core/lowres_clock.hh>
 #include <seastar/core/print.hh>
 
 #include <optional>
 #include <ostream>
 
+inline ss::sstring human_readable_bytes(uint64_t bytes) {
```

Comment:
we alredy thave this implemented
in the code and used widely.

Reply:
here is the code, we don't need this file at all. https://github.com/vectorizedio/redpanda/blob/dev/src/v/utils/human.cc

Reply:
noice